server.py

```diff
@@ -0,0 +1,84 @@
+"""
```

Comment:
Hi, the reason why I put the 2 issues into one commit is so that I can check that the Vagrantfile actually works. Otherwise it will just be a Vagrantfile that I'm not sure if this file works or not. Still thanks for the advice!